### DISCUSSION TITLE
Is this library going to be maintained?

### DISCUSSION THREAD

**Discussion by nxorable** [scikit-optimize/scikit-optimize#1161] [General]
I see a lot of PRs, but the library has had no updates in quite some time. It's a good library otherwise.

**Comment by gregorkschroeder** [comment]
This seems to be an issue with a couple informed hyper-parameter search libraries. Both scikit-optimize and hyperopt appear to be abandoned. Maybe use optuna with the ray-tune package for a potential drop-in like replacement?
